### DISCUSSION TITLE
Request higher image display quality

### DISCUSSION THREAD

**Discussion by VictoryLuode** [zsviczian/obsidian-excalidraw-plugin#1175] [Ideas]
These are two pictures. The top is the source picture (resolution of 5000*5000), and the bottom is the picture in Excalidraw.

However, after importing to Excalidraw, the image quality is obviously reduced, which affects the user experience to a certain extent. I hope the developer can improve this problem. I try to replace PureRef with Excalidraw.|

I really like Excalidraw, hope you heed my request, thanks!
![001](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/60289531/1d5fad26-6808-4a07-a708-5c0b50e409e1)

**Comment by Rez-Shok** [comment]
You can first import the file to your vault then insert it to excalidraw. inserting a picture directly into the drawing cause quality reduce I hope developers fix this problem.

**Reply by CMCDragonkai** [reply]
Where is this vault?

**Reply by Luk3Pl4ys** [reply]
He is talking about your Obsidian vault. It's just your workspace where all your files live.
